StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -476,7 +476,7 @@ extension STPAPIClient: FinancialConnectionsAPIClient {
             .lowercased()
         body["phone_number"] = phoneNumber
         body["country"] = country
-        body["locale"] = (phoneNumber != nil) ? Locale.current.identifier : nil
+        body["locale"] = (phoneNumber != nil) ? Locale.current.toLanguageTag() : nil
```

Comment:
[optional parameter](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/networking/methods/link_account_session_save_to_link_method.rb#L42)

Reply:
little trickier to analyze where its going, but regardless its doubtful it handles iOS locale's